.eslintrc

```diff
@@ -16,20 +16,54 @@
         "__SCREENSHOT__": false
     },
     "plugins": [
-        "flow-vars", "mocha", "filenames"
+        "flow-vars", "mocha", "filenames", "babel", "import"
     ],
+    "settings": {
+        "import/core-modules": [ electron, react-native ],
+        "import/resolver": {
+          webpack: 'webpack.config.base.js'
+        },
+    },
     "rules": {
+        "babel/flow-object-type": [1, "comma"],
+        "babel/func-params-comma-dangle": 0,
+        "comma-dangle": [2, "always-multiline"],
+        "filenames/match-regex": [2, "^[0-9a-z-.]+(\\.desktop|\\.native|\\.ios|\\.android)?$"],
```

Comment:
:sheep: Old code, but wanted to note that this doesn't seem to have any real effect, since the (.desktop|.native|etc) match is covered by the preceding `$[0-9a-z-.]+`.